core/integration/features/verified/LP-large-prob-trading-bug.feature

```diff
@@ -0,0 +1,105 @@
+    #   | id  | decimal places |
+    #   | USD | 18             |
+    And the fees configuration named "fees-config-1":
+      | maker fee | infrastructure fee |
+      | 0.004     | 0.001              |
+    And the price monitoring named "price-monitoring-1":
+      | horizon | probability | auction extension |
+      | 10000   | 0.99        | 300               |
+    And the markets:
+      | id        | quote name | asset | risk model              | margin calculator         | auction duration | fees          | price monitoring   | oracle config          |
+      | ETH/MAR22 | ETH        | USD   | log-normal-risk-model-1 | default-margin-calculator | 1                | fees-config-1 | default-none | default-eth-for-future |
+    And the parties deposit on asset's general account the following amount:
+      | party  | asset | amount    |
+      | party0 | USD   | 5000000000   |
+      | party1 | USD   | 10000000000 |
+      | party2 | USD   | 10000000000 |
+      | party3 | USD   | 10000000000 |
+
+    Given the following network parameters are set:
+      | name                                          | value |
+      | market.stake.target.timeWindow                | 24h   |
+      | market.stake.target.scalingFactor             | 1     |
+      | market.liquidity.bondPenaltyParameter         | 0.2   |
+      | market.liquidity.targetstake.triggering.ratio | 0.1   |
+      | market.liquidity.stakeToCcySiskas             | 1.0   |
+
+   And the average block duration is "1"
+
+   Scenario: LP vol seems wrong when a new best bit/ask price comes in
+
+   And the parties submit the following liquidity provision:
+      | id  | party  | market id | commitment amount | fee   | side | pegged reference | proportion | offset | lp type    |
+      | lp1 | party0 | ETH/MAR22 | 50000             | 0.001 | sell | ASK              | 500        | 1      | submission |
+      | lp1 | party0 | ETH/MAR22 | 50000             | 0.001 | buy  | BID              | 500        | 1      | amendment  |
+
+    And the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     | reference  |
+      | party1 | ETH/MAR22 | buy  | 1      | 1     | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1  |
+      | party1 | ETH/MAR22 | buy  | 1      | 890   | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1  |
+      | party1 | ETH/MAR22 | buy  | 1      | 900   | 0                | TYPE_LIMIT | TIF_GTC | buy-ref-1  |
+      | party2 | ETH/MAR22 | sell | 3      | 900   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-1 |
+      | party2 | ETH/MAR22 | sell | 1      | 990   | 0                | TYPE_LIMIT | TIF_GTC | sell-ref-2 |
+
+    When the opening auction period ends for market "ETH/MAR22"
+    Then the auction ends with a traded volume of "1" at a price of "900"
+    # target_stake = mark_price x max_oi x target_stake_scaling_factor x rf = 1000 x 10 x 1 x 0.1
```

Comment:
Nitpick: comment is misleading if the mark price is 900 and open interest is 1 (rather than 1000 and 10).